src/cli/simple/command/mod.rs

```diff
@@ -1069,3 +1069,33 @@ impl ContractCall<'_> {
         Ok(())
     }
 }
+
+pub async fn add_relayer(
+    context: Context,
+    account_id: String,
+    deposit: f64,
```

Comment:
Using float types for money, it's a bad idea.

```suggestion
    deposit: u128,
```